components/BottomSheet.tsx

```diff
@@ -21,15 +23,18 @@ import Animated, {
   useAnimatedStyle,
   useSharedValue
 } from "react-native-reanimated"
+import { FullWindowOverlay } from "react-native-screens"
 
 export const TiFBottomSheetProvider = BottomSheetModalProvider
 
 type BaseTiFBottomSheetProps = {
   sizing: TiFBottomSheetSizing
+  overlay?: "on-screen" | "above-screen"
```

Comment:
So having this overlay prop is strictly for iOS? I also wasn't fully sure what it did; I'm guessing it's to have it appear above a component?

Reply:
Yes, because on iOS the sheet would appear behind modals presented with the native stack navigator. `"above-screen"` ensures that it always appears above any view.

And it is strictly for iOS, as `FullWindowOverlay` does not work on Android.